src/app/store.ts

```diff
@@ -1,6 +1,10 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
```

Comment:
The import statement uses combineSlices, which is incorrect. It should be combineReducers from '@reduxjs/toolkit'. Please update this to ensure the reducers are combined correctly.